backendAS/pg.sql

```diff
@@ -0,0 +1,35 @@
+-- TEST
+DROP TABLE if exists testsong;
+DROP TABLE if exists testcomment;
+
+CREATE TABLE testsong(
+  id int,
+  songname varchar(80),
```

Comment:
recommend analyzing your data seed generator to encapsulate the largest `varchar` length it outputs.  In SQL, column widths are fixed and take up extra harddisk space, so if you aren't using all 80, it will consume more HD.  In practice, this is a balance between forward planning (i.e. largest possible character length) versus disk savings.

Reply:
Explicit, it's unlikely you have a song title of 80 characters and it becomes `# of rows` * `# of unused character spaces` which becomes a very large number.